org.scala-ide.sdt.core/src/org/scalaide/ui/internal/editor/ScalaToggleBreakpointAdapter.scala

```diff
@@ -144,7 +151,7 @@ class ScalaToggleBreakpointAdapter extends ToggleBreakpointAdapter with HasLogge
    * The implementation of this method is copied from the super class, which had
    * to be overwritten because it doesn't know how to get a ScalaCompilationUnit.
    */
-  override def translateToMembers(part: IWorkbenchPart, selection: ISelection): ISelection = {
+  def translateToMembers(part: IWorkbenchPart, selection: ISelection): ISelection = {
```

Comment:
This is potentially problematic, because some of the `ToggleBreakpointAdapter` methods will now call original method which was made `static`.